docs/getting-started.md

```
@@ -0,0 +1,152 @@
+    bun run build           # produces dist/agentsync (compiled binary)
+    ./dist/agentsync --version
+    ```
+
+You also need a private Git repository to use as the **vault remote** —
+GitHub, GitLab, Gitea, or any reachable Git server. The age keypair
+that protects the vault contents is generated for you on first `init`;
+you do not need to install the `age` CLI separately.
+
+## 2. Create the vault on Machine A
+
+```bash
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)